src/actions/websocket.test.js

```diff
@@ -365,6 +365,33 @@ describe('Actions.Websocket', () => {
         test();
     });
 
+    it('Websocket Handle Channel Unarchive', (done) => {
+        async function test() {
```

Comment:
Instead of the inner `test`, could you just run this whole test as an async function?
```javascript
it('Websocket Handle Channel Unarchive', async (done) => {
});
```